acceptance/tests/api-gateway/api_gateway_test.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	StaticClientName           = "static-client"
-	gatewayClassControllerName = "hashicorp.com/consul-api-gateway-controller"
+	gatewayClassControllerName = "consul.hashicorp.com/gatewayclass-controller"
```

Comment:
I know this is technically what's being targeted, but thinking it'd probably make more sense to use the shorter name since this is something users would need to add manually if creating their own `GatewayClass` objects:

```suggestion
	gatewayClassControllerName = "consul.hashicorp.com/gateway-controller"
```

Reply:
Did this with a find/replace so I'll close your suggestion.